dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/config/internal/WebFragmentsInfoImpl.java

```diff
@@ -61,13 +63,11 @@ class WebFragmentsInfoImpl implements WebFragmentsInfo {
         this.servletSpecLevel = servletSpecLevel;
 
         // Web app ...
-
         WebApp webApp = containerToAdapt.adapt(WebApp.class); // throws UnableToAdaptException
         if (webApp != null) {
             this.servletSchemaLevel = webApp.getVersion();
-            if ("6.0".equals(servletSchemaLevel) || "5.0".equals(servletSchemaLevel) || "4.0".equals(servletSchemaLevel) || "3.1".equals(servletSchemaLevel)
-                || "3.0".equals(servletSchemaLevel)
-                || "2.5".equals(servletSchemaLevel)) {
+            int servletSchemaLevelInt = PlatformVersion.getVersionInt(servletSchemaLevel);
```

Comment:
Instead of on PlatformVersion class shouldn't this be on WebApp class and be specific to servlet?  Otherwise we would not catch invalid versions.  11.0 isn't valid for web app version right now for instance.

Reply:
Also the getVersionInt makes "6" a valid value and I am not sure if that is meant to be a valid value for the version.